llama_stack/apis/agents/agents.py

```diff
@@ -541,3 +559,32 @@ async def delete_agent(
         :param agent_id: The ID of the agent to delete.
         """
         ...
+
+    @webmethod(route="/agents", method="GET")
+    async def list_agents(self) -> ListAgentsResponse:
+        """List all agents.
+
+        :returns: A ListAgentsResponse.
+        """
+        ...
+
+    @webmethod(route="/agents/{agent_id}", method="GET")
+    async def get_agent(self, agent_id: str) -> AgentConfig:
```

Comment:
Thanks for the update! To align with `list_agents`?
```suggestion
    async def get_agent(self, agent_id: str) -> Agent:
```